src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -688,6 +836,25 @@ const ViewPopup = (() => {
             .append($('<div/>', { id: 'notice_ongoing', class: `fill-bg-color gr-padding-10 ${hidden_class}`, text: localize('You can close this window without interrupting your trade.') }));
 
         ViewPopupUI.showInpagePopup(`<div class="${popupbox_id}">${$container.html()}</div>`, '', '#sell_bet_desc');
+        // Pass all contract details to new quill contract detail popup
+        // cd_ prefix is Contract Details
+        purchaseManager.set({
+            cd_description     : longcode,
+            cd_showEntrySpot   : should_show_entry_spot,
```

Comment:
can we choose either snake or camel case please? or do we have a guideline for a mixed camel+snake case?)

Reply:
@maryia-deriv in quill we are following came case but in smarttrader it's mostly snake case.

Reply:
this requires a prefix so that the data is grouped accordingly. Can we add exemption in this case?